baseplate/observers/timeout.py

```diff
@@ -43,10 +43,21 @@ def __init__(self, timeout_config: config.ConfigNamespace):
 
     def on_server_span_created(self, context: RequestContext, server_span: ServerSpan) -> None:
         timeout = self.config.by_endpoint.get(server_span.name, self.config.default)
+
+        min_timeout = None
         if timeout is not config.InfiniteTimespan:
-            observer = TimeoutServerSpanObserver(
-                server_span, timeout.total_seconds(), self.config.debug
-            )
+            min_timeout = timeout.total_seconds()
+        try:
```

Comment:
nit: can you put a blank line before this so it's disconnected from the preceding `if` block please?